src/mount.js

```diff
@@ -7,8 +7,10 @@ import createInstance from './lib/create-instance'
 import cloneDeep from 'lodash/cloneDeep'
 import createElement from './lib/create-element'
 import './lib/matches-polyfill'
+import { setVueErrorHandler } from './lib/error-handler'
 
 Vue.config.productionTip = false
+setVueErrorHandler(Vue)
```

Comment:
I think we should export errorHandler and call Vue.config.errorHandler = errorHandler, instead of calling a function. Especially since we set productionTip false just above the function

Reply:
Hey, on this one we are going to need to do this config.errorHandler in each place that we have an entry into vue-test-utils (shallow, mount and create local vue)  will update with individual imports

Side thought for now or future, it may make sense to have a custom version of Vue that we import from so that we can isolate any global things like this to a single point.  

```import Vue from '.lib/base-vue'```